source/postprocess/geoid.cc

```diff
@@ -957,13 +1048,13 @@ namespace aspect
                             Patterns::Bool(),
                             "Option to also output the spherical harmonic coefficients of the geoid anomaly up to the maximum degree. "
                             "The default is false, so postprocess will only output the geoid anomaly in grid format. ");
-          prm.declare_entry("Also output the spherical harmonic coefficients of surface dynamic topography contribution", "false",
+          prm.declare_entry("Also output the spherical harmonic coefficients of surface topography contribution", "false",
                             Patterns::Bool(),
-                            "Option to also output the spherical harmonic coefficients of the surface dynamic topography contribution "
+                            "Option to also output the spherical harmonic coefficients of the surface topography contribution "
                             "to the maximum degree. The default is false. ");
-          prm.declare_entry("Also output the spherical harmonic coefficients of CMB dynamic topography contribution", "false",
+          prm.declare_entry("Also output the spherical harmonic coefficients of CMB topography contribution", "false",
                             Patterns::Bool(),
-                            "Option to also output the spherical harmonic coefficients of the CMB dynamic topography contribution "
+                            "Option to also output the spherical harmonic coefficients of the CMB topography contribution "
```

Comment:
Changing the names of input parameters will break every input file that uses them, which is why we do this very rarely. Here are two options: 
(1) Leave them unchanged and accept the `dynamic` in the parameter. 
(2) Rename them, and create an alias for the old parameter (`prm.declare_alias(new_parameter_name, old_parameter_name);`). If you decide to go this route, we should come up with better names. E.g. `Output geoid anomaly coefficients`, `Output surface topography contribution coefficients`, `Output CMB topography contribution coefficients`, `Output density anomaly contribution coefficients`, and `Output gravity anomaly coefficients`.
I leave the choice between (1) and (2) up to you.

Reply:
Im going for 2 :)